orchestration/dags/data_gcp_dbt/models/intermediate/firebase/pro/int_firebase__pro_event.sql

```diff
@@ -0,0 +1,75 @@
+{% set target_name = target.name %}
+{% set target_schema = generate_schema_name('analytics_dbt_' ~ target_name) %}
+
+{{ config(
+    pre_hook="{{create_dehumanize_id_function()}}"
+) }}
+
+{{
+    config(
+        materialized = "incremental",
+        incremental_strategy = "insert_overwrite",
+        partition_by = {"field": "event_date", "data_type": "date"},
+        on_schema_change = "sync_all_columns"
+    )
+}}
+
+
+WITH pro_event_raw_data as(
+    SELECT 
+        event_name,
+        user_pseudo_id,
+        CASE WHEN REGEXP_CONTAINS(user_id, r"\D") THEN {{target_schema}}.dehumanize_id(user_id) ELSE user_id END AS user_id,
+        platform,
+        event_date,
+        event_timestamp,
+        ga_session_number as session_number,
+        ga_session_id as session_id,
+        CONCAT(user_pseudo_id, '-',ga_session_id) as unique_session_id,
+        origin,
+        destination,
+        traffic_campaign,
+        traffic_medium,
+        traffic_source,
+        category as user_device_category,
+        operating_system as user_device_operating_system,
+        operating_system_version as user_device_operating_system_version,
+        browser as user_web_browser,
+        browser_version as user_web_browser_version,
+        COALESCE(CAST(offerer_id as STRING),CAST(offererId AS STRING)) as offerer_id,
+        COALESCE(venue_id,CAST(venueId as STRING)) as venue_id,
+        page_location,
+        page_title as page_name,
+        REGEXP_EXTRACT(page_location, r"""passculture\.pro\/(.*)$""", 1) as url_first_path,
+        REGEXP_EXTRACT(page_location, r"""passculture\.pro\/(.*?)[\/.*?\/.*?\/|\?]""", 1) as url_path_type,
+        REGEXP_EXTRACT(page_location, r"""passculture\.pro\/.*?\/.*?\/(.*?)\/|\?""", 1) as url_path_details,
+        REGEXP_EXTRACT(page_location, r"""passculture\.pro\/(.*?)\?""", 1) as url_path_before_params,
+        REGEXP_EXTRACT_ALL(page_location,r'(?:\?|&)(?:([^=]+)=(?:[^&]*))') as url_params_key,
+        REGEXP_EXTRACT_ALL(page_location,r'(?:\?|&)(?:(?:[^=]+)=([^&]*))') as url_params_value,
+        REGEXP_REPLACE(REGEXP_REPLACE(page_location , "[A-Z \\d]+[\\?\\/\\&]?", ""), "https://passculture.pro/", "") as url_path_agg,
+        page_referrer,
+        page_number,
+        COALESCE(double_offer_id,offerId) AS offer_id,
+        offerType as offer_type,
+        used,
```

Comment:
We could rename theses columns to be a bit clearer on what the might be (used/saved/ eac_wrong_student_modal_only6and5)